.pre-commit-config.yaml

```diff
@@ -76,3 +76,9 @@ repos:
     hooks:
       - id: flake8
         name: Check PEP8
+
+  - repo: https://github.com/asottile/yesqa
+    rev: v1.2.3
+    hooks:
+      - id: yesqa
+        name: Unused noqa
```

Comment:
Since this modified code and can impcat pep8, this hook should run before isort, black and flake8